package.json

```diff
@@ -69,21 +69,21 @@
     "format-fix": "prettier './packages/**/*.ts' --write"
   },
   "devDependencies": {
+    "@eslint/js": "^9.3.0",
     "@remusao/auto-config": "^1.1.2",
     "@types/benchmark": "^2.1.3",
     "@types/node": "^20.6.2",
-    "@typescript-eslint/eslint-plugin": "^7.0.1",
-    "@typescript-eslint/parser": "^7.0.1",
     "auto": "^11.0.5",
     "benchmark": "^2.1.4",
     "chalk": "^5.3.0",
-    "eslint": "^8.49.0",
+    "eslint": "^9.3.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-prettier": "^5.0.0",
     "lerna": "^8.0.0",
     "patch-package": "^8.0.0",
     "prettier": "^3.0.3",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "typescript-eslint": "8.0.0-alpha.16"
```

Comment:
Not stable yet?

Reply:
on the way, but we can confirm it works https://github.com/typescript-eslint/typescript-eslint/issues/8211